src/components/PoolInfoCard.tsx

```diff
@@ -78,8 +72,9 @@ function PoolInfoCard({ data }: Props): ReactElement {
           {formattedData.tokens.map((token, index) => (
             <div className="token" key={index}>
               <img alt="icon" src={token.icon} />
-              <span>{token.name}</span>
-              <span className="tokenPercent">{token.percent}%</span>
+              <span className="bold">
+                {token.name + " " + token.percent + "%"}
```

Comment:
prefer interpolation for this whole component `{'${token.name} ${token.percent}%'}`